dvc/stage/serialize.py

```diff
@@ -134,18 +134,18 @@ def to_pipeline_file(stage: "PipelineStage"):
 def to_single_stage_lockfile(stage: "Stage") -> dict:
     assert stage.cmd
 
+    def _dumpd(item):
+        ret = [
+            (item.PARAM_PATH, item.def_path),
+            *item.hash_info.to_dict().items(),
+        ]
+
+        return OrderedDict(ret)
```

Comment:
This should eventually turn into a special Dumper for outputs or into `Output.dumpd_lock`(or smth like that). We've discussed it before.